advanced/usage-with-module-loaders/reference/skins.md

```diff
@@ -0,0 +1,135 @@
+</span><span class="k"><span class="kwd">import</span></span><span class="pln"> </span><span class="s1"><span class="str">'tinymce/skins/ui/example/skin.css'</span></span><span class="p"><span class="pun">;</span></span><span class="pln">
+</span><span class="cm"><span class="com">/* Only required for inline editors */</span></span><span class="pln">
+</span><span class="k"><span class="kwd">import</span></span><span class="pln"> </span><span class="nx"><span class="pln">contentUiInlineCss</span></span><span class="pln"> </span><span class="k"><span class="kwd">from</span></span><span class="pln"> </span><span class="s1"><span class="str">'tinymce/skins/ui/example/content.inline.css'</span></span><span class="p"><span class="pun">;</span></span><span class="pln">
+</span><span class="cm"><span class="com">/* Only required when using the editor in a shadowDOM */</span></span><span class="pln">
+</span><span class="k"><span class="kwd">import</span></span><span class="pln"> </span><span class="s1"><span class="str">'tinymce/skins/ui/example/skin.shadowdom.css'</span></span><span class="p"><span class="pun">;</span></span>
+</code><button class="copy-to-clipboard-button">COPY CODE</button></pre></div></div>
+</td>
+</tr>
+<tr>
+<td><code>.zip</code>&nbsp;</td>
+<td>
+<div class="language-js highlighter-rouge"><div class="highlight"><pre class="prettyprint prettyprinted" style=""><code><span class="k"><span class="kwd">import</span></span><span class="pln"> </span><span class="nx"><span class="pln">contentUiCss</span></span><span class="pln"> </span><span class="k"><span class="kwd">from</span></span><span class="pln"> </span><span class="s1"><span class="str">'../tinymce/skins/ui/example/content.css'</span></span><span class="p"><span class="pun">;</span></span><span class="pln">
+</span><span class="k"><span class="kwd">import</span></span><span class="pln"> </span><span class="s1"><span class="str">'../tinymce/skins/ui/example/skin.css'</span></span><span class="p"><span class="pun">;</span></span><span class="pln">
+</span><span class="cm"><span class="com">/* Only required for inline editors */</span></span><span class="pln">
+</span><span class="k"><span class="kwd">import</span></span><span class="pln"> </span><span class="nx"><span class="pln">contentUiInlineCss</span></span><span class="pln"> </span><span class="k"><span class="kwd">from</span></span><span class="pln"> </span><span class="s1"><span class="str">'../tinymce/skins/ui/example/content.inline.css'</span></span><span class="pln">
+</span><span class="cm"><span class="com">/* Only required when using the editor in a shadowDOM */</span></span><span class="pln">
+</span><span class="k"><span class="kwd">import</span></span><span class="pln"> </span><span class="s1"><span class="str">'../tinymce/skins/ui/example/skin.shadowdom.css'</span></span><span class="p"><span class="pun">;</span></span>
```

Comment:
I don't believe this is correct... shadow dom likely requires some other crazy configuration. Reason being the skin.css won't be able to be imported like this when using shadow dom because it's effectively within a new frame/window and would need to be remotely loaded. At this stage I don't think you'd be able to use a module loader that does bundling with ShadowDOM.

Reply:
https://stackoverflow.com/a/63959654 Is the sort of thing that would be required for the skins.css file to get shadowdom working.

Reply:
Fixed in [`da26917` (#2035)](https://github.com/tinymce/tinymce-docs/pull/2035/commits/da269174535a702a4c727fb6074d0e15349bd409)